### PR TITLE
[FX] Make some modifications to limitation section

### PR DESCRIPTION
**Pull request by Chillee**

### CODE REVIEW

File: docs/source/fx.rst [hunk 1]
```diff
@@ -818,11 +818,7 @@ symbolic tracing:
 
         fx.symbolic_trace(f) # Fails!
 
-        def wrapper(flag):
-            return lambda x: f(x, flag)
-
-        new_f = wrapper(flag=True)
-        fx.symbolic_trace(new_f)
+        fx.symbolic_trace(f, concrete_args={'flag': True})
```
Comment (jamesr66a) [inline docs/source/fx.rst:821]: I'd rework the lead-in sentence above the code example to mention something like "or by binding concrete values to arguments during symbolic tracing"

File: docs/source/fx.rst [hunk 2]
```diff
@@ -968,7 +964,17 @@ Miscellanea
...
+    ::
+
+        @torch.fx.wrap
+        def torch_randn(x, shape):
+            return torch.randn(shape)
+
+        def f(x):
+            return x + torch_randn(x, 5)
+        fx.symbolic_trace(f)
+
```
Comment (jamesr66a) [inline docs/source/fx.rst:977]: Does this render properly in the context of the bulleted list?

Comment (Chillee) [reply docs/source/fx.rst:977]: yeah I have a screenshot.

**Review by jamesr66a — APPROVED** [review]
make sure to cherrypick to the `release/1.8` branch and link your PR in the release issue https://github.com/pytorch/pytorch/issues/51886

**Pull request closed without merging**